app/public/css/application.css

```diff
@@ -15,7 +15,7 @@ video {
 video::-webkit-media-controls-fullscreen-button {
   display: none;
 }
-
+/* can't test that one locally but this is the place where the decision logo is styled @paulactivecinema */
```

Comment:
Hey Robert. yeah you're right, but actually that was not supposed to get merged into the master. Paul just asked me whether I could have a look where these settings might be set. that's what I was doing - I just forgot to delete this PR right after that.